docs/developers/releasing.md

```diff
@@ -0,0 +1,106 @@
+# Releasing
+Karmada has two types of releases, major release, and minor release. For example, `v1.3.0` is a major release, and `v1.3.1` is a minor release. The relationship between release, tag, and branch is as follows:
```

Comment:
It's weird to say `xxx has two types of releases`.

Reply:
In addition, v1.3.0 is a minor release, and v1.3.1 is a patch release.

Reply:
fixed